modules/brushingandlinking/src/datastructures/indexlist.cpp

```diff
@@ -50,13 +52,20 @@ const BitSet& IndexList::getIndices() const {
     return indices_;
 }
 
-void IndexList::set(std::string_view src, const BitSet& indices) {
+bool IndexList::set(std::string_view src, const BitSet& indices) {
+    if ((indicesBySource_.count(std::string(src)) > 0)) {
+        if (indicesBySource_[std::string(src)] == indices) return false;
```

Comment:
here you are constructing the string twice and doing the same lookup in the map twice? 
```suggestion
    if (auto it = indicesBySource_.find(std::string(src)): it != indicesBySource_.end()) {
        if (it->second == indices) return false;
```

Reply:
and again below...